File: distribution/RealmBasicExample/app/src/main/java/io/realm/examples/realmintroexample/model/Dog.java
package io.realm.examples.realmintroexample.model;

import io.realm.RealmObject;

public class Dog extends RealmObject {
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: examples/coroutinesExample/src/main/java/io/realm/examples/coroutinesexample/model/Dog.kt
package io.realm.examples.coroutinesexample.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Dog : RealmObject() {
    @PrimaryKey
    var name: String = ""
    var age: Int = 1
    var owner: String = ""

    override fun toString(): String = "Name: $name, age: $age, owner: $owner"
}
